libs/base/Data/String.idr

```diff
@@ -1,5 +1,6 @@
 module Data.String
 
+import Data.String.Extra
```

Comment:
This looks like a dependency of `base` to `contrib` which should not be. I think it's better to move `join` function from `contrib` to `base` for that case.

Reply:
Big +1 from me on moving `join` from contrib to base.

Reply:
https://github.com/idris-lang/Idris2/blob/54332752f0cb43645f365d6780a00c3348a4377e/libs/base/Data/String.idr#L41-L52

How's this?